packages/build-electron/src/start/index.ts

```diff
@@ -8,27 +9,28 @@ import {
     BuildEvent
 } from '@angular-devkit/architect';
 import { normalize, getSystemPath } from '@angular-devkit/core';
+import { WebpackBuilder } from '@angular-devkit/build-webpack';
 
 import { Observable, of, merge, TeardownLogic } from 'rxjs';
 import { concatMap, share, first, switchMap, dematerialize, materialize } from 'rxjs/operators';
 
 import { ElectronStartBuilderSchema } from './schema';
-import { WebpackBuilder } from '@angular-devkit/build-webpack';
 
 export class ElectronStartBuilder implements Builder<ElectronStartBuilderSchema> {
 
     constructor(public context: BuilderContext) { }
 
     run(builderConfig: BuilderConfiguration<ElectronStartBuilderSchema>): Observable<BuildEvent> {
-        const targetString = builderConfig.options.browserTarget;
-        const [project, target, configuration] = targetString.split(':');
+        const browserTarget = builderConfig.options.browserTarget.replace(':serve', ':build');
+        const overrides = { ...builderConfig.options, ...{ browserTarget } };
```

Comment:
You don't need to spread the browserTarget and instead of asserting and then deleting the webpackConfig key, you could do something like that:

```typescript
  const { webpackConfig, ...overrides } = { ...builderConfig.options, browserTarget };
```